src/world/area_kkj/kkj_19/AF7C00.c

```diff
@@ -4,29 +4,273 @@
 
 #include "world/common/AddPlayerHandsOffset.inc.c"
 
+extern s32 D_80241B10_AF9460;
```

Comment:
Please put all externs in the file together at the top of the file before the function declarations, ordered by memory - we prefer to keep them all together there